lua/fiber: do not raise on printing a dead fiber (#9099)
tarantool/tarantool: changelogs/unreleased/gh-4265-printing-a-finished-fiber-stops-the-program.md

## bugfix/core

> * Now, an error is not raised on the attempt to print a dead fiber (gh-4265).

Reviewer: ```suggestion
* An error is no longer raised on an attempt to print a dead fiber (gh-4265).
```

Author: Fixed.
